### ISSUE TITLE
path variable support and new log file detection

### ISSUE THREAD

**Issue by jiejisheng** [elastic/logstash-forwarder#481]
[Backgroud:]

The nginx log files located at .../%{YEAR}/%{MONTH}, and the a new log file would be created with name %{YY.MM.DD}-access.log everyday.

[Requirement:]
1. Variable Support in path
2. With globs or path, new file detections would be nice instead of restarting the service

**Comment by lmorfitt** [comment]
@jiejisheng have you tried the path formatting of /var/logs/nginx/*/*/*.log ?

**Comment by jiejisheng** [comment]
@lmorfitt , Yup, the issue is that this log directory won't be cleaned up, so it contains logs of year 2013 or even earlier. To avoid the unnecessary file monitor, the accurate path/file naming format would reduce the cost.

**Comment by lmorfitt** [comment]
@jiejisheng Are, I see. Personally I would logrotate (with compress) the older files after 30 days (or something sane), thus LSF will not monitor .gz files

**Comment by angel-smile** [comment]
I have got the same problem with log-courier. So this hasn't been resolved in logstash-forwarder either?

@jiejisheng Can I know what's your workaround at the moment?
